src/Usuario.js

```diff
@@ -21,10 +21,10 @@ class Usuario {
     }   
 
     /**
-     * Método que devuelve las rutas del usuario
-     * @returns {Array} rutas Rutas del usuario
+     * Método que devuelve la informacion de las rutas del usuario
+     * @param {Array[Map]} infoRutas Informacion de las Rutas realizadas por el usuario
```

Comment:
¿Dónde dice aquí que la información en inforutas es la que has "propuesto" en #23? No se trata de que propongáis cosas, se trata de que analicéis y escribáis código. Este cambio que has hecho en realidad, dado que no aporta información adicional, no sirve de nada. Funcionalmente es un cambio de nombre, lo único que has cambiado es la documentación (y eso también es incorrecto)